src/styles.less

```diff
@@ -52,6 +52,7 @@ body {
   user-select: none;
   overscroll-behavior: none;
   height: 100%;
+  width: 100%;

```

Comment:
@immortalityidle Double check that this doesn't screw up the main panel. Possibly some sizing/scrolling capability you were relying on got busted. Same with line 97

Reply:
I'll take a look as soon as I have time.